content/kubeflow/training.md

```diff
@@ -39,13 +38,30 @@ If you want to use an existing bucket in a different region, then make sure to s
 
 AWS credentials are required to save model on S3 bucket. These credentials are stored in EKS cluster as Kubernetes secrets.
 
-Get your AWS access key id and secret access key.
+Create an IAM user 's3user', attach S3 access policy and retrieve temporary credentials
+```
+aws iam create-user --user-name s3user
+aws iam attach-user-policy --user-name s3user --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
+aws iam create-access-key --user-name s3user| tee /tmp/create_output.json
+```
 
-Replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in the following command with values specific to your environment.
+You will get similar output
+```
+{
+	"AccessKey": {
+		"UserName": "s3user",
+		"Status": "Active",
+		"CreateDate": "2019-11-08T00:53:25Z",
+		"SecretAccessKey": < AWS Secret Access Key > ,
+		"AccessKeyId": < AWS Access Key >
+	}
+}
+```
+Replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in the following command with values received from above output.
```

Comment:
Instructions are a bit confusing. They require to replace `AWS_ACCESS_KEY_ID` where as it should be `REPLACE_WITH_AccessKeyId`. Ditto for other too